src/chains/ethereum/address/index.ts

```diff
@@ -1,14 +1,15 @@
-import { Data, JsonRpcDataInputArg } from "@ganache/utils";
+import { Address as EJSAddress } from "@ethereumjs/util";
+import { JsonRpcDataInputArg, Data } from "@ganache/utils";
 
-export class Address extends Data {
+export class Address extends EJSAddress {
   static ByteLength = 20;
 
-  constructor(value: string | Buffer) {
-    super(value, Address.ByteLength);
+  constructor(value: Buffer) {
```

Comment:
> Why change the signature of the function? Wouldn't it be nice to be able to new Address("0x123123123123");?
I don't have particularly strong opinions on this, but to me it seemed to make sense for the constructor to match the `EJSAddress` that we are extending, and the `.from` function do the conversions. If the `.from` only calls `new Address`, why does it need to exist?

> Additionally, this throws if the length of value is not 20 - do we want to allow different length buffers to be passed?
I would rather not allow different length buffers to be passed because: 
1. we didn't have any existing cases where this happened
2. I don't want to increase the performance overhead of this change if it isn't necessary